demos/python_demos/multi_camera_multi_person_tracking/multi_camera_multi_person_tracking.py

```diff
@@ -141,13 +179,28 @@ def main():
     log.info("Creating Inference Engine")
     ie = IECore()
 
-    person_detector = Detector(ie, args.m_detector, args.t_detector,
-                               args.device, args.cpu_extension,
-                               capture.get_num_sources())
+    if args.detections:
+        person_detector = DetectionsFromFileReader(args.detections, args.t_detector)
+    else:
+        if args.m_segmentation:
```

Comment:
`elif args.m_segmentation:` would be more readable.

Reply:
Fixed.